packages/wallets/src/components/WalletsActionScreen/WalletsActionScreen.tsx

```diff
@@ -38,13 +38,9 @@ const WalletsActionScreen: React.FC<PropsWithChildren<TProps>> = ({
                         {title}
                     </WalletText>
                 )}
-                {isValidElement(description) ? (
-                    description
-                ) : (
-                    <WalletText align='center' size={descriptionSize}>
-                        {description}
-                    </WalletText>
-                )}
+                <WalletText align='center' size={descriptionSize}>
+                    {description}
+                </WalletText>
```

Comment:
If we will distinguish based on `isValidElement` we are loosing `align='center' size={descriptionSize}` parameters that centering the text so we are using WalletText wrapper only once. I have checked and it works for strings and `<Localize />` components. Please shout if any concerns.